helm/nginx-ingress/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+# @klotho::execution_unit { 
+# id = "nginx-helm"
+# }
+apiVersion: v2
+appVersion: 2.4.1
+description: NGINX Ingress Controller
+home: https://github.com/nginxinc/kubernetes-ingress
+icon: https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v2.4.1/deployments/helm-chart/chart-icon.png
+keywords:
+- ingress
+- nginx
+kubeVersion: '>= 1.19.0-0'
+maintainers:
+- email: kubernetes@nginx.com
+  name: nginxinc
+name: nginx-ingress
+# @klotho::embed_assets {
+#   id = "static-assets"
+#   include = "nginx-ingress/templates/_helpers.tpl"
+# }
```

Comment:
nice! I think we shouldn't rely on the yaml files getting copied over and just do `templates/*` to make sure

We can discuss later what files get included in the unit or not. If we do tree-shaking (and since nothing references yaml files) we might lose these from the unit.

Reply:
yup, commented that i needed to add readme, but wanted to publish so it was clear how this worked with the pr thats out. 

I think making them specify embed assets isnt nice for the UX, but youre right with the tree shaking so well need to have a plan for that

Reply:
> I think making them specify embed assets isnt nice for the UX

Agreed, we should be able to be more intelligent as we add more first-class Helm support.